packages/chart/src/Column.ts

```diff
@@ -280,11 +280,8 @@ export class Column extends XYAxis {
                     if (context.showInnerText()) {
                         _texts
                             .text((d: any) => {
-                                let innerText = d.row[0];
+                                const innerText = context.innerText(d.origRow, d.origRow[columnLength], d.idx);
                                 if (innerText) {
-                                    if (innerText instanceof Array) {
-                                        innerText = innerText[d.idx - 1];
-                                    }
```

Comment:
@jbrundage - why was this code needed?

Reply:
It allowed for custom labels for each series in a category (when dealing with ND data)

Reply:
@jbrundage Can you show me an example (as I removed this code?)